memo/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+
+# Note: This crate must be built using do.sh
+
+[package]
+name = "spl-memo"
+version = "0.1.0"
+description = "Solana Program Library Memo"
+authors = ["Solana Maintainers <maintainers@solana.com>"]
+repository = "https://github.com/solana-labs/solana-program-library"
+license = "Apache-2.0"
+edition = "2018"
+
+[dependencies]
+solana-sdk = { version = "=1.2.0", default-features = false, features=["program"] }
+solana-sdk-bpf-test = { path = "../bin/bpf-sdk/rust/test", default-features = false }
```

Comment:
Just looking at this name here, with the `test` suffix. Can this be put in `[dev-dependencies]`?

Reply:
Ah, actually memo doesn't need that one at all either